tabs/mixer.js

```diff
@@ -333,22 +333,51 @@ TABS.mixer.initialize = function (callback, scrollPosition) {
             content: $('#mixerApplyContent')
         });
 
-        $('#execute-button').click(function () {
-            helper.mixer.loadServoRules(currentMixerPreset);
-            helper.mixer.loadMotorRules(currentMixerPreset);
+        heliModal = new jBox('Modal', {
+            width: 480,
+            height: 240,
+            closeButton: 'title',
+            animation: false,
+            title: chrome.i18n.getMessage("heliModalTitle"),
+            content: $('#heliModalContent')
+        });
+
+        function mixerApply() {
+            heliModal.close();
+            heliSettings = {
+                pitchRollWeight: $('#pitch-roll-weight').val(),
+                useTailMotor: $('#use-tail-motor').is(':checked')
+            };
+            helper.mixer.loadServoRules(currentMixerPreset, heliSettings);
+            helper.mixer.loadMotorRules(currentMixerPreset, heliSettings);
             renderServoMixRules();
             renderMotorMixRules();
             renderOutputMapping();
-            modal.close();
+        }
+
+        function mixerApplyAndReboot() {
+            mixerApply();
             saveAndReboot();
+        }
+
+        function mixerApplyWithModal(apply) {
+            $('#pitch-roll-weight').val(50);
```

Comment:
Why this is set to 50 over here? Should it not be reseted to current value?

Reply:
@DzikuVx the reason for this is that this setting only exists as a "virtual" setting to calculate mixer weight. Will be changed if this kind of mixing is handled on FC side https://github.com/iNavFlight/inav/pull/4714)

Reply:
That's what I remembered from FC pull requests. And that's why I'm asking. I'd prefer not to merge to configurator something that is not working on FC code. And also prefer to have configurator pull requests as complete as they can be. The way I see it, right now it's dead code that does not work

Reply:
So, TL;DR is that FC Heli support should be complete and ready. Makes sense - at the moment FC pull request does not allow heli mixer to be flyable AFAU.

Reply:
@digitalentity  is right. It's not flyable for now, but it is disabled for now, so noone can try to fly it.
From your discussion I understand that I should complete all I need for helicopter to be flyable, then make some test flights, debug that as much as I can, commit code to these two branches (inav and inav-configurator). Am I right?